tests/tests.js

```diff
@@ -81,23 +74,23 @@ define(['jquery', 'zimArchive', 'zimDirEntry', 'util', 'utf8'],
      * @param e Error
      */
     function errorHandlerAsyncTest(e) {
-        ok(false, "Error in async call", e);
-        start();
+        assert.ok(false, "Error in async call", e);
+        done();
     }
```

Comment:
This is a missing piece wrt error handling that I haven't figured out yet. Every time something goes wrong in a test nothing shows up on the page. There is no indication even that the test was run. QUnit seems to swallow up all errors.